client/src/components/NewPastaForm.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+
+const NewPastaForm = ({ postPasta }) => {
+  const [newPasta, setNewPasta] = useState({
+    name: "",
+    description: "",
+  });
+
+  const handleInputChange = (event) => {
+    setNewPasta({
+      ...newPasta,
+      [event.currentTarget.name]: event.currentTarget.value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    newPasta.name.toLowerCase();
+    postPasta(newPasta);
+    clearForm();
```

Comment:
What happens if the form does not submit successfully? Does the form still clear? Or do we want to ensure that the form only clears when submitted successfully so that the user doesn't lose any previous data they supplied